wolfcrypt/src/port/maxim/maxq10xx.c

```diff
@@ -600,6 +600,9 @@ static int aes_set_key(Aes* aes, const byte* userKey, word32 keylen)
         goto end_AesSetKey;
     }
 
+    /* Delete object if one already exists. Ignore error in case it doesn't
+     * exist. */
+    mxq_rc = MXQ_DeleteObject(obj_id);
```

Comment:
Static analysis will not like this. Instead do `(void)MXQ_DeleteObject(obj_id);`.